x-pack/plugins/fleet/common/types/rest_spec/agent_policy.ts

```diff
@@ -83,3 +83,13 @@ export interface GetFullAgentConfigMapResponse {
 export interface GetFullAgentManifestResponse {
   item: string;
 }
+
+export type AgentPolicyClientFetchAllAgentPoliciesOptions = Pick<
+  ListWithKuery,
+  'perPage' | 'kuery' | 'sortField' | 'sortOrder'
+> & { fields?: string[] };
+
+export type AgentPolicyClientFetchAllAgentPolicyIdsOptions = Pick<
```

Comment:
nit: these new types look quite verbose compared to the others, I think it would be more elegant to leave the `AgentPolicyClient` prefix. what do you think?

Reply:
Removed